### PR TITLE
Ecommerce card

### PR DESCRIPTION
**Pull request by RahullRawat**
Screenshot (93)
Screenshot (94)

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]

File: components/cards/cards.css [hunk 1]
```diff
@@ -61,6 +61,82 @@
...
 	top: 2%;
 }
+
+.product-card {
+	width: 24rem;
+	margin-top: 5rem;
+	border-radius: 0.5rem;
+}
+
+.product-card > a > img {
```
Comment (Jay-Gadhiya) [inline components/cards/cards.css]: I suggest don't use selectors, instead use the direct class names for a particular element. 
ex. : .product-card ( which is first)
        .product-card-link ( you use '> a' )
        .product-card-img (you use '> img')

File: components/cards/cards.html [hunk 2]
```diff
@@ -266,6 +266,73 @@ <h1>Card Dismiss</h1>
...
+								><img src="../../assets/diary5.jpg" alt="paintings"
+							/></a>
+
+							<div class="price-container text-center">
+								<p class="item-name">Sun Painting</p>
+								<span class="currency">Rs 999.00</span>
+							</div>
+							<a href="#"
+								><button class="btn btn-primary">Add to cart</button></a
+							>
```
Comment (Jay-Gadhiya) [inline components/cards/cards.html]: you can improve code indentation, if it's default by format document then after format document you can manually refactor.

File: components/cards/cards.html [hunk 3]
```diff
@@ -266,6 +266,73 @@ <h1>Card Dismiss</h1>
...
+							style="
+								width: 682px;
+								height: 428px;
+								border: 0;
+								transform: scale(1);
+								overflow: hidden;
+								margin-top: 2rem;
```
Comment (Jay-Gadhiya) [inline components/cards/cards.html:335]: instead of inline style, you can use class of it.

**Review by Jay-Gadhiya — APPROVED** [review]
everything looks good, good work.

**Pull request merged**